tests/core_tests.py

```diff
@@ -438,5 +438,11 @@ def test_extra_table_metadata(self):
             '/caravel/extra_table_metadata/{dbid}/'
             'ab_permission_view/panoramix/'.format(**locals()))
 
+    def test_process_template(self):
```

Comment:
could you please add test to the sql_lab tests demonstrating how it works end to end ?